lib/parser.ts

```diff
@@ -29,15 +29,39 @@ export const getDueDate = (issue: Pick<GithubIssueDataWithChildren, 'html_url' |
   };
 };
 
-function getSectionLines(text: string, sectionHeader: string): string {
-  const sectionIndex = text.indexOf(sectionHeader);
-  if (sectionIndex === -1) {
+function regexIndexOf(string: string, regex: RegExp, startpos = 0) {
+  const indexOf = string.substring(startpos).search(regex);
+  if (indexOf >= 0) {
+    return indexOf + startpos
+  }
+  return indexOf;
+}
+
+function indexOf(string: string, strOrRegex: string | RegExp, startpos = 0) {
+  if (typeof strOrRegex === 'string') {
+    return string.indexOf(strOrRegex, startpos)
+  }
+  return regexIndexOf(string, strOrRegex, startpos)
+}
+
+function getSectionLines(text: string, sectionHeader: string | RegExp): string {
+  const sectionStartIndex = indexOf(text, sectionHeader);
+  if (sectionStartIndex === -1) {
     return '';
   }
-  return text.substring(sectionIndex)
+  const startText = text.substring(sectionStartIndex)
+  /**
+   * sectionEndIndex marks the location of the first double line break
+   * i.e. first empty line
+   */
+  const sectionEndIndex = regexIndexOf(startText, /^[\r\n]{2,}$/gm)
+  if (sectionEndIndex === -1) {
+    return startText
+  }
+  return startText.substring(0, sectionEndIndex)
```

Comment:
This change prevents an error where this function was including "\r\n\r\nMilestone: https://github.com/filecoin-station/core/milestone/4" in the response. 

blank lines should be recognized as the end of a "section"